src/OrchardCore.Modules/OrchardCore.Autoroute/Handlers/AutoroutePartHandler.cs

```diff
@@ -109,6 +109,7 @@ public override async Task UpdatedAsync(UpdateContentContext context, AutorouteP
                 templateContext.SetValue("ContentItem", part.ContentItem);
 
                 part.Path = await _liquidTemplateManager.RenderAsync(pattern, NullEncoder.Default, templateContext);
+                part.Path = part.Path.Replace("\r\n", String.Empty);
```

Comment:
I merged, but I should have checked, this won't work on linux. You should replace both `\r` and ``\n` individually.

Reply:
I will do a quick direct fix then